plugins/Kaleidoscope-Redial/src/kaleidoscope/plugin/Redial.h

```diff
@@ -20,7 +20,7 @@
 #include "kaleidoscope/Runtime.h"
 #include <Kaleidoscope-Ranges.h>
 
-#define Key_Redial Key(kaleidoscope::ranges::REDIAL)
+constexpr Key Key_Redial = kaleidoscope::Key(kaleidoscope::ranges::REDIAL);
```

Comment:
The "kaleidoscope::Key" in this commit doesn't match the patterns in other similar header file changes. Was that intentional?

Reply:
No, it wasn't.  Thanks.  It is, of course, the same thing, since `::Key` is a typedef of `::kaleidoscope::Key`, but I _meant_ to do all of them in the same style.